views/index.jade

```diff
@@ -0,0 +1,10 @@
+extends ./layout.jade
+
+block content
+  main
+    ul
+      -
+        links = [['/ron', 'Ron Swanson'], ['/dog', 'Dog Slap'], ['/banana', 'Banana Dance'], ['/bees', 'Not The Bees']]
+      each item in links
```

Comment:
I would define the `links` array above the `ul`, trying to keep variables towards the top will make it easier to read in the long run.